arviz/plots/pairplot.py

```diff
@@ -68,13 +76,29 @@ def plot_pair(
         Matplotlib axes or bokeh figures.
     divergences_kwargs : dicts, optional
         Additional keywords passed to ax.scatter for divergences
-    plot_kwargs : dicts, optional
-        Additional keywords passed to ax.plot, az.plot_kde or ax.hexbin
+    scatter_kwargs:
```

Comment:
Given the changes parts of the docstring are outdated. For example 

    kind : str
        Type of plot to display (scatter, kde or hexbin)

is not an "or" but and/or. And probably we should explicitly indicate the overlaying feature.

Instead of
`Plot a scatter or hexbin matrix of the sampled parameters.`

we should have something like:
 
Scatter, kde and/or hexbin matrix with (optional) marginals on the diagonal

Reply:
Also the examples need to be updated. It's OK for me if we do that in another PR (but we should open and issue so we do not forget).